Makefile

```
@@ -126,6 +126,10 @@ fmt: ## Run go fmt against code.
 vet: ## Run go vet against code.
 	go vet ./...
 
+.PHONY: lint
+lint: ## Run golangci-lint against code.
+	golangci-lint run
+
```

Running it after the first checkout results in

```
❯ make lint
golangci-lint run
bash: line 1: golangci-lint: command not found
make: *** [Makefile:131: lint] Error 127
```

I think it would be good to follow the principle of least surprise, so let `make` download the binary to the `LOCALBIN` if not present.

There are also a few tweaks for the linter execution itself worth applying:

```suggestion
.PHONY: lint
lint: ## Run golangci-lint against code.
	golangci-lint run --fix --sort-results
```